web/src/pages/index.tsx

```diff
@@ -1,56 +1,64 @@
-import { Img, Text, Heading, Box, SimpleGrid, Stack, Center, Flex } from '@chakra-ui/react'
-import { FaBook, FaChalkboardTeacher, FaUserGraduate, FaQuestion } from 'react-icons/fa'
+import { Img, Text, Heading, Box, SimpleGrid, Flex } from '@chakra-ui/react'
+import { FaBook, FaChalkboardTeacher, FaUserGraduate, FaQuestion, FaCalendarWeek } from 'react-icons/fa'
 import { PageScaffold } from '../components/PageScaffold'
 import WithAuth, { WithAuthProps } from '../components/withAuth'
 import { LinkBoxHomePage } from '../components/LinkBoxHomePage'
 import React from 'react'
 import source from '../images/Logo_GeMont.png'
+import { Role } from '../generated/graphql'
+import { IconType } from 'react-icons'
+
+interface BoxLink {
+  icon: IconType,
+  url: string,
+  text: string,
+  roles: Role[]
+}
 
 const Index: React.FC<WithAuthProps> = ({ self }) => {
-  const getTeacherLinkBoxes = () => {
+  const arrayOfLinkBoxes: BoxLink[] = [
+    {
+      icon: FaChalkboardTeacher,
+      url: '/tutorium',
+      text: 'Tutorium Management',
+      roles: [Role.Teacher, Role.Coordinator]
+    },
+    {
+      icon: FaUserGraduate,
+      url: '/student',
+      text: 'Schüler Management',
+      roles: [Role.Teacher, Role.Coordinator]
+    },
+    {
+      icon: FaBook,
+      url: '/absence',
+      text: 'Abwesenheit buchen',
+      roles: [Role.Teacher, Role.Coordinator]
+    },
+    {
+      icon: FaCalendarWeek,
+      url: '/semester',
+      text: 'Zeitspanne erstellen',
+      roles: [Role.Coordinator]
+    },
+    {
+      icon: FaQuestion,
+      url: 'https://lam-gemont.github.io/fehlzeiten/',
```

Comment:
kann sich dieser Link in einem neuen Tab öffen?

Reply:
Ich schaue mal;). Wäre auf jeden Fall sinnvoll ^^